File: app/src/main/java/ru/visualmath/android/api/model/Page.java
package ru.visualmath.android.api.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Page implements Serializable {

    @SerializedName("index")
    private int index;

    @SerializedName("type")
    private String type;

    public int getIndex() {
        return index;
    }

    public String getType() {
        return type;
    }
}

File: app/src/main/java/ru/visualmath/android/api/model/Page.kt
package ru.visualmath.android.api.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Page(

        @SerializedName("index")
        val index: Int = 0,

        @SerializedName("type")
        val type: String = ""
) : Parcelable
